test/run_test_confluent.sh

```diff
@@ -100,8 +100,8 @@ case $CONFLUENT_VERSION in
 	6.2.*)
     DOWNLOAD_URL="https://packages.confluent.io/archive/6.2/confluent-community-$CONFLUENT_VERSION.tar.gz"
     ;;
-  7.2.*)
-    DOWNLOAD_URL="https://packages.confluent.io/archive/7.2/confluent-community-$CONFLUENT_VERSION.tar.gz"
+  7.6.*)
+    DOWNLOAD_URL="https://packages.confluent.io/archive/7.6/confluent-community-$CONFLUENT_VERSION.tar.gz"
```

Comment:
I have a strong feeling that we can do better than this :)

Reply:
Should we add a ticket for tech debt?

Reply:
I doubt if we ever have enough space to tackle such low priority issues. I will try to fix it by myself in the next PR rather than create a ticket.